src/mbgl/layout/symbol_layout.cpp

```diff
@@ -301,12 +303,20 @@ void SymbolLayout::addFeature(const SymbolFeature& feature,
             }
         }
     } else if (feature.type == FeatureType::Polygon) {
-        // TODO: pole of inaccessibility
-        for (const auto& ring : feature.geometry) {
-            for (const auto& point : ring) {
-                Anchor anchor(point.x, point.y, 0, minScale);
-                addSymbolInstance(ring, anchor);
+        for (const auto& polygon : classifyRings(feature.geometry)) {
+            Polygon<double> poly;
+            for (const auto& ring : polygon) {
+                LinearRing<double> r;
+                for (const auto& p : ring) {
+                    r.push_back(convertPoint<double>(p));
+                }
+                poly.push_back(r);
             }
+
+            // 16 here represents 2 pixels
```

Comment:
Ok; can we replace this with the symbolic constants that we're using throughout GL native?